### PR TITLE
Workout Logger

### PR DESCRIPTION
**Pull request by mkewls**
** Merge conflict in main.jsx b/c of fix to onEnter in earlier-in-time merge
** This is a fully stand-alone page view, but can be modified into a component, as desired
** To visually test, pay attention to what needs to be commented-in and -out in main.jsx

### CODE REVIEW

**Comment by mkewls** [comment]
** Also, forgot to note that the majority of the workout reducer is empty, and reserved for our eventual dashboard redux

File: app/components/counter/Counter.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                onClick={ () => this.onPainClick(5) }
+              >
+                <img src={require("../../../src/images/emojis/5pain.svg")} style={ emoji } />
+              </IconButton>
+            </div>
+            <div style={ center }>
+              <TextField
+                hintText="Any notes from the workout?"
+                floatingLabelText="Comments"
+                value={ this.state.comments }
```
Comment (jakepeyser) [inline app/components/counter/Counter.js]: I don't think this value is needed, the field will work without it passed back down

**Comment by jakepeyser** [comment]
A few minor comments, but after you resolve conflict with `/main.js`, LGTM

File: app/components/counter/Counter.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { logActivity } from '../../reducers/workouts'
+// material ui
+import { CircularProgress, Drawer, FloatingActionButton, IconButton, Paper, RaisedButton, TextField } from 'material-ui'
```
Comment (mkewls) [inline app/components/counter/Counter.js]: Yup

**Pull request merged**